### PR TITLE
The commit of the century

### PR DESCRIPTION
**Pull request by gerhardengelbrecht**
There might not have been any commits (code) 20 years ago but this feels like it would make the top 5000.

### CODE REVIEW

**Comment by gerhardengelbrecht** [comment]
How do I get rid of education/GitHubGraduation-2021 on my GitHub account?

**Comment by asifrasheed6** [comment]
Goto settings and delete the repo 😇

**Comment by gerhardengelbrecht** [comment]
open says me

File: README.md [hunk 1]
```diff
@@ -33,14 +33,14 @@ Information submitted to [the swag shipment form](https://airtable.com/shrM5IigB
 
 Replace `<YOUR-USERNAME>` with your GitHub username in this guide.
 
-### First, create the folder _data/YOUR-USERNAME/ 
-Fork this repository, create a new folder inside the `_data` folder, and name it with your username. It should look something like this `_data/<YOUR-USERNAME>/`. Ex.
```
Comment (gerhardengelbrecht) [inline README.md:37]: Chose this one

File: README.md [hunk 2]
```diff
@@ -33,14 +33,14 @@ Information submitted to [the swag shipment form](https://airtable.com/shrM5IigB
...
+### First, create the folder _data/gerhardengelbrecht/ 
+Fork this repository, create a new folder inside the `_data` folder, and name it with your username. It should look something like this `_data/<gerhardengelbrecht>/`. Ex.
 
 ```
 _data/MonaTheOctocat/
 ```
 ### Second, add your profile information
-Create a markdown file in your folder following the convention `<YOUR-USERNAME>.md`. Ex.
```
Comment (gerhardengelbrecht) [inline README.md:43]: Chose this one

**Comment by gerhardengelbrecht** [comment]
Open

**Pull request closed without merging**